intermol/gromacs/gromacs_parser.py

```diff
@@ -59,7 +59,7 @@ def default_gromacs_include_dir():
     if 'GMXBIN' in os.environ:
         return os.path.abspath(os.path.join(
             os.environ['GMXBIN'], '..', 'share', 'gromacs', 'top'))
-    return '/usr/local/gromacs/share/gromacs/top'
+    return '/usr/local/share/gromacs/top'
```

Comment:
I'll check later tonight but if that was indeed changed moving to 5.x then we should add a check for if the directory exists and prefer 5.x over 4.x